components/src/utils/Bloomreach.js

```diff
@@ -0,0 +1,35 @@
+/**
```

Comment:
We're using typescript now so this should be a ts file

Reply:
Sounds good i can change them to .ts.  Noticed that there are two utils folders in both components and app.  The files in components are all .js files so I will change those as well.  Not sure if we are needing both folders though.  Might need to open a seperate issue for that.